src/quotes.ts

```diff
@@ -63,5 +63,51 @@ export default [
     "Dear frozen yogurt, you are the celery of desserts. Be ice cream or be nothing. Zero stars.",
     "There has never been a sadness that can’t been cured by breakfast food.",
     "If any of you need anything at all, too bad. Deal with your problems yourselves, like adults.",
-    "It’s pointless for a human to paint scenes of nature when they can go outside and stand in it."
-];
+    "It’s pointless for a human to paint scenes of nature when they can go outside and stand in it.",
+    "On my deathbed, my final wish is to have my ex-wives rush to my side so I can use my dying breath to tell them both to go to hell one last time.",
+    "I was born ready. I’m Ron F*cking Swanson.",
+    "Don’t start chasing applause and acclaim. That way lies madness.",
+    "If there were more food and fewer people, this would be a perfect party.",
+    "Put some alcohol in your mouth to block with words from coming out.",
+    "Give 100%. 110% is impossible. Only idiots recommend that.",
+    "Birthdays were invented by Hallmark to sell cards",
```

Comment:
@jamesseanwright - I have missed the `.` at the end of this quote. Can you please edit this file directly so that I don't open new pull request just because of this?

Reply:
Just fixed this and deployed! https://ron-swanson-quotes.herokuapp.com/v2/quotes/search/birthdays